tests/e2e/specs/shopper/checkout-single-use-coupon.test.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * Internal dependencies
+ */
+import { shopper } from '../../../utils';
+import { createCoupon } from '../../utils';
+import { SIMPLE_PRODUCT_NAME } from '../../../utils/constants';
+import { getOrderDetailsDiscountPathExpression } from '../../../utils/path-expressions';
+
+if ( process.env.WOOCOMMERCE_BLOCKS_PHASE < 2 )
+	// eslint-disable-next-line jest/no-focused-tests
+	test.only( `Skipping checkout tests`, () => {} );
+
+let couponCode;
+
+beforeAll( async () => {
+	couponCode = await createCoupon( { usageLimit: 1 } );
```

Comment:
We usually clear the setup in the afterAll branch, should we also remove this coupon that was created?

Reply:
Thank you for pointing this out, @ralucaStan! Now, we have `deleteCoupon()` in the `afterAll` block.